js/js_value.cc

```diff
@@ -401,7 +401,7 @@ std::string from_js(const JSValue & val, std::string *)
     }
     else
     {
-    	return *v8::String::AsciiValue(val);
```

Comment:
That's not right.  We're now assuming that all std::strings have UTF8 in them.  Instead, we should use the override for UtfString, and make sure that we correctly pass that through.